Warning: no access to tty (Bad file descriptor).
Thus no job control in this shell.
/home1/datahome/xyu/.miniconda3/envs/equinox/lib/python3.6/site-packages/distributed/utils.py:128: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  % (host, default, e), RuntimeWarning)
distributed.nanny - INFO -         Start Nanny at: 'tcp://10.148.0.133:49895'
distributed.nanny - INFO -         Start Nanny at: 'tcp://10.148.0.133:36861'
/home1/datahome/xyu/.miniconda3/envs/equinox/lib/python3.6/site-packages/distributed/utils.py:128: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  % (host, default, e), RuntimeWarning)
distributed.worker - INFO -       Start worker at:   tcp://10.148.0.133:38558
distributed.worker - INFO -          Listening to:   tcp://10.148.0.133:38558
distributed.worker - INFO -              nanny at:         10.148.0.133:36861
distributed.worker - INFO -              bokeh at:          10.148.0.133:8789
/home1/datahome/xyu/.miniconda3/envs/equinox/lib/python3.6/site-packages/distributed/utils.py:128: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  % (host, default, e), RuntimeWarning)
distributed.worker - INFO -       Start worker at:   tcp://10.148.0.133:42926
distributed.worker - INFO - Waiting to connect to:   tcp://10.148.0.253:52734
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -          Listening to:   tcp://10.148.0.133:42926
distributed.worker - INFO -              nanny at:         10.148.0.133:49895
distributed.worker - INFO -              bokeh at:         10.148.0.133:54470
distributed.worker - INFO - Waiting to connect to:   tcp://10.148.0.253:52734
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -               Threads:                          4
distributed.worker - INFO -                Memory:                   50.00 GB
distributed.worker - INFO -       Local Directory: /dev/shm/pbs.2021360.datarmor0/worker-rwetmoqb
distributed.worker - INFO -               Threads:                          4
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -                Memory:                   50.00 GB
distributed.worker - INFO -       Local Directory: /dev/shm/pbs.2021360.datarmor0/worker-rzhjdu53
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -         Registered to:   tcp://10.148.0.253:52734
distributed.worker - INFO - -------------------------------------------------
distributed.core - INFO - Starting established connection
distributed.worker - INFO -         Registered to:   tcp://10.148.0.253:52734
distributed.worker - INFO - -------------------------------------------------
distributed.core - INFO - Starting established connection
distributed.core - INFO - Event loop was unresponsive in Worker for 4.55s.  This is often caused by long-running GIL-holding functions or moving large chunks of data. This can cause timeouts and instability.
distributed.core - INFO - Event loop was unresponsive in Worker for 4.55s.  This is often caused by long-running GIL-holding functions or moving large chunks of data. This can cause timeouts and instability.
distributed.worker - INFO - Stopping worker at tcp://10.148.0.133:38558
distributed.worker - INFO - Stopping worker at tcp://10.148.0.133:42926
distributed.worker - INFO - Connection to scheduler broken.  Reconnecting...
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - Connection to scheduler broken.  Reconnecting...
distributed.worker - INFO - -------------------------------------------------
distributed.nanny - INFO - Closing Nanny at 'tcp://10.148.0.133:36861'
distributed.nanny - INFO - Closing Nanny at 'tcp://10.148.0.133:49895'
distributed.dask_worker - INFO - End worker
/home1/datahome/xyu/.miniconda3/envs/equinox/lib/python3.6/site-packages/distributed/utils.py:128: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  % (host, default, e), RuntimeWarning)
